l10n_it_fatturapa_in/wizard/wizard_import_fatturapa.py

```diff
@@ -645,9 +645,9 @@ def _createPayamentsLine(self, payment_id, line, partner_id):
                                     'name': dline.IstitutoFinanziario,
                                     'bic': dline.BIC,
                                 }
-                            ).id
+                            )
                     else:
-                        bankid = banks[0].id
+                        bankid = banks[0]
```

Comment:
coglierei l'occasione per rinominare `bankid` in `bank`.

Reply:
Fatto in https://github.com/OCA/l10n-italy/compare/a5bdc24584e6eb7945f1860ca3cc69d26bba1d8a..10a43c18cb9b4121adf551fe9f9116269040bf9f

Reply:
Grazie.